app/controllers/home_controller.rb

```diff
@@ -1,5 +1,6 @@
 class HomeController < ApplicationController
   def index
     redirect_to account_posts_path if current_user
+    @posts = Post.all.ordered.limit(24)
```

Comment:
I see that you have limit number 24 in all your queries, please move it to some constant or set in configs if it's possible